scripts/mapnik/c7d29b826/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode9.3
```

Comment:
Bump this to `xcode11.3` to get a recent SDK (9.3 is pretty old at this point): https://docs.travis-ci.com/user/reference/osx/#macos-version

Reply:
great, thanks!

Reply:
done in 5a8450bf88a9d75e48a83d3abe016a3914d05384 
btw, current mapnik is `b22d5d2fa`